src/bundles/SellerRegistration/components/ProductsForm/ProductsForm.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import { actions } from 'react-redux-form';
+
+import ProductsForm, { mapStateToProps } from './ProductsForm';
+
+import createStore from '../../redux/create'
+
+
+const generateFormValidilityState = (valid) => {
+    return {
+        forms: {
+            productsForm: {
+                $form: { valid }
+            }
+        },
+        form_options: {
+            mode: 'add'
+        }
+    }
+}
+
+test('mapStateToProps', () => {
+    let state = generateFormValidilityState(true);
+    let props = mapStateToProps(state);
+    expect(props).toEqual({ form: {"valid": true}, "formErrors": undefined, "mode": "add", "model": "productsForm", "returnLink": undefined});
+
+    state = generateFormValidilityState(false);
+    props = mapStateToProps(state);
+    expect(props).toEqual({ form: { valid: false },"formErrors": undefined, "mode": "add", "model": "productsForm", "returnLink": undefined});
+});
```

Comment:
These tests are redundant, since the selector (formProps) you're using already has a test